DOCUMENTATION.md

```diff
@@ -2827,6 +2827,43 @@ for await (const row of sqlResult) {
 }
 ```
 
+The "json-flat" value format means that top level fields of the json object is treated as separate columns. There is also "json"
+value format which we cover in the next section.
+
+### 8.7.7. JSON Data Type
+
+JSON is a first-class SQL type with IMap and Kafka Connector support, usable in most contexts, similarly to any other type.
+
+In order to send a JSON parameter use a `HazelcastJsonValue` object:
+
+```javascript
+await client.getSql().execute('INSERT INTO test VALUES (2, ?)', [new HazelcastJsonValue(JSON.stringify({age: 3}))]);
+```
+
+The corresponding result type for JSON type is `HazelcastJsonValue` as well:
+
+```javascript
+const result = await client.getSql().execute('SELECT * FROM test');
+
+const rowMetadata = result.rowMetadata;
+rowMetadata.getColumn(rowMetadata.findColumn('this')).type.should.be.eq(SqlColumnType.JSON);
```

Comment:
Is this a test? `type.should.be.eq` will not be available for the most of the users